core-bundle/contao/templates/_new/frontend_module/template.html.twig

```diff
@@ -1 +0,0 @@
-{% extends "@Contao/content_element/template.html.twig" %}
```

Comment:
Was this file removed on purpose?

Reply:
Yes, I did not see, why it would be relevant anymore?

Reply:
`frontend_module/template.html.twig` is the template for the _Template_ module.

Reply:
Oh, I thought that was one I added during the process 😂 Will restore